VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -540,8 +540,10 @@ def get_wala_version_from_command(self):
             while(process_wait_time > 0 and p.poll() is None):
                 time.sleep(1)
                 process_wait_time -= 1
-            out = p.stdout.read()
+            out = p.stdout.read().decode()
```

Comment:
as mentioned offline, can you just check in BCM that whether we are throwing warning when version deserialization is failing

Reply:
The version() throwing below error..
Exception in IsUnsupportedGuestAgentVersion System.ArgumentException: Version string portion was too short or too long.
   at System.Version.VersionResult.SetFailure(ParseFailureKind failure, String argument)
   at System.Version.TryParseVersion(String version, VersionResult& result)
   at System.Version.Parse(String input)
   at System.Version..ctor(String version)
   at Microsoft.Internal.CloudBackup.Service.Common.ExtensionManager.ExtensionManagerHelper.IsUnsupportedGuestAgentVersion(String osType, String guestAgentVersion) in X:\bt\1213499\repo\src\Service\ExtensionManager\ExtensionMgrHelper.cs:line 559